packages/core/src/amazonqFeatureDev/client/featureDev.ts

```diff
@@ -230,6 +238,18 @@ export class FeatureDevClient {
                     e.code === 'ServiceQuotaExceededException')
             ) {
                 throw new CodeIterationLimitError()
+            } else if (
+                isAwsError(e) &&
+                e.name === 'ValidationException' &&
```

Comment:
For `CodewhispererRuntimeException`, we will have to use `e.code` for verifying exception type.